edge_computing/image-base-install/ibi-understanding-image-based-install.adoc

```diff
@@ -45,14 +61,35 @@ During this process, the `openshift-install` program installs {op-system-first}
 [id="ibi-deployment-overview"]
 === Image-based deployment for {sno} clusters
 
-You can use the `openshift-install` program to configure and deploy a host that you preinstalled with an image-based installation. To configure the target host with site-specific details, you must create the following resources:
+You can use the `openshift-install` program or the IBI Operator to configure and deploy a host that you preinstalled with an image-based installation.
+
+{sno} cluster deployment::
+
+To configure the target host with site-specific details by using the `openshift-install` program, you must create the following resources:
 
 * The `install-config.yaml` installation manifest
 
 * The `image-based-config.yaml` manifest
 
 The `openshift-install` program uses these resources to generate a configuration ISO that you attach to the preinstalled target host to complete the deployment.
 
+Managed {sno} cluster deployment::
+
+{rh-rhacm-first} and the multicluster engine for Kubernetes operator (MCE) use a hub-and-spoke architecture to manage and deploy {sno} clusters across multiple sites. Using this approach, the hub cluster serves as a central control plane that manages the spoke clusters, which are often remote {sno} clusters deployed at the far edge of the network.
```

Comment:
I think this is because in the ACM docs, they do not capitalize the word "Operator" as we do in OCP. See here: https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html-single/multicluster_global_hub/index#global-hub-operator. Even the [official RH product name list](https://docs.google.com/spreadsheets/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/edit?gid=1375785039#gid=1375785039) uses the noncapitalized form